src/components/UI/MainInput.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="wrapper">
+    <label :for="iputCat" class="input-label"><slot /></label>
```

Comment:
I don't think this is going to work 🤔 
Label's `for` attribute is used to target the input with a given `id`,
i.e.
```html
<label for="name">...</label>
<input id="name" type="text" />
```
Otherwise this won't have any effect so we might remove the `for` attribute, or try to figure out how to bind an id to the input and also place it as the `for` attribute's value 😄

Reply:
thank you so much for this!!! I created another prop to take input name and bound it to label for and input id - hope this is a much better solution 🎉